ui/preloader/preloader.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+    `application`
+}
+
+application {
+    mainClassName = "edu.wpi.grip.preloader.GripPreloader"
+}
+
+tasks.named<JavaExec>("run") {
+    classpath = sourceSets["main"].runtimeClasspath
+    main = application.mainClassName
+    args = listOf("windowed")
```

Comment:
Windowed?

Reply:
For launching the preloader by itself. See the preloader application class